service/dap/server.go

```diff
@@ -1049,6 +1071,32 @@ func (s *Session) onLaunchRequest(request *dap.LaunchRequest) {
 		return
 	}
 
+	func() {
+		if args.Console == "internalConsole" && runtime.GOOS == "linux" && (args.Backend == "default" || args.Backend == "native") {
```

Comment:
It's weird that using internalConsole on an unsupported configuration fails silently. It's also weird that all values of console that aren't internalConsole are accepted as valid.

Reply:
The current behavior is that `console` is not checked, so it can also be completely invalid and `dlv dap` would accept it, so I'm not sure we necessarily have to validate the `console` argument.

Reply:
Suppose someone misspells internlaConsole, how do they notice?

Reply:
Usually, such syntax can be checked in VSCode itself - it can present an error such as: `"internalConsole" is not a valid "console" value".

Reply:
Not everyone uses vscode with dap. I thought that was the whole point of this.

Reply:
Hm, true. I am not familiar with how every DAP client handles this, but adding an allowed list of declared DAP consoles (I think it is standardized) should be easy enough.

Reply:
it should just be the consoles that we support.